src/shared/slices/publicationsSlice.ts

```diff
@@ -0,0 +1,34 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from '../rootReducer';
+
+type PublicationListElement = {
+    id: number, 
+    dirPath: string, 
+    publicationName: string
+    };
+
+type PublicationList = {
+        list: [PublicationListElement]
+    };
+
+
+  const initialState: PublicationList = {list:[{id: 0, dirPath:"", publicationName: ""}]};
```

Comment:
why is this PublicationList an object?

Reply:
I did't know how to export the list without making it an object, so I could export the "list" field.